main/src/main/scala/sbt/Defaults.scala

```diff
@@ -123,7 +123,7 @@ object Defaults extends BuildCommon {
       retrieveManagedSync :== false,
       configurationsToRetrieve :== None,
       scalaOrganization :== ScalaArtifacts.Organization,
-      sbtResolver := { if (sbtVersion.value endsWith "-SNAPSHOT") Classpaths.typesafeSnapshots else Classpaths.typesafeReleases },
+      sbtResolver := Classpaths.typesafeReleases,
```

Comment:
sbt is now published to Maven Central for the releases, we might keep using https://bintray.com/sbt/ivy-snapshots for the nightly.